### ISSUE TITLE
Automated JWT fuzzing

### ISSUE THREAD

**Issue by andresriancho** [cle0patra/burp-jwt-extension#4]
Maybe you want to use https://github.com/andresriancho/jwt-fuzzer to provide automated JWT fuzzing?

**Comment by pinnace** [comment]
@andresriancho 

Yes, definitely! This is great, I was looking for a tool like this to generate test cases for the ActiveScanner. I will be investigating further 👍

**Comment by andresriancho** [comment]
From and active scan perspective I believe that it is safe to say that: every fuzzed JWT should trigger a 401  (or whatever page you get when the JWT is invalid), and if that doesn't happen, then a vulnerability was found.

Vulnerabilities can range from exceptions to auth bypass. If the modified JWT returns the same response as the original one, most likely an auth bypass (or the JWT is not even used in that endpoint).

**Comment by pinnace** [comment]
I think that is a fair assumption. I think I'd also record an invalid baseline, maybe by flipping some bits in the signature of the valid JWT and recording response code + content hash and using that as separate baseline to catch other errors.
